### DISCUSSION TITLE
Difference between 'iou-thresh' and 'conf-thresh'

### DISCUSSION THREAD

**Discussion by ASSO-MDSE-MoDLF** [ultralytics/yolov5#7904] [Q&A] [answered]

**Comment by pourmand1376** [comment, marked as answer]
These are two separate thresholds to convert your `positive` predictions into `negative`. YOLO outputs with each bbox a confidence score. If that confidence score is below `conf-thres`, that image is ignored. Actually that bbox doesn't show up in results and it is considered as nothing. This is important since in some scenarios, a prediction is very important, so model should be really sure about that. 

IOU-thres is also similar, however this time we calculate the `Intesection-over-union` and filter out results based on that. 

Read [this](https://www.mdpi.com/1424-8220/21/13/4350/htm) paper for more explanation on `conf-thres`.

**Reply by PandemicModelingWorkbench** [reply]
Thanks for your response!
